package.json

```
@@ -1,31 +1,25 @@
 {
   "name": "react-quill",
   "version": "1.3.3",
-  "description": "The Quill rich-text editor as a React component.",
+  "description": "The Quill rich-text editor as a React component",
   "author": "zenoamaro <zenoamaro@gmail.com>",
-  "homepage": "https://github.com/zenoamaro/react-quill",
   "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/zenoamaro/react-quill/issues",
-    "email": "zenoamaro@gmail.com"
-  },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/zenoamaro/react-quill.git"
-  },
+  "homepage": "https://github.com/zenoamaro/react-quill",
+  "bugs": "https://github.com/zenoamaro/react-quill/issues",
+  "repository": "https://github.com/zenoamaro/react-quill.git",
   "main": "lib/index.js",
-  "browser": "lib/index.js",
-  "types": "types.d.ts",
+  "types": "lib/index.d.ts",
   "scripts": {
-    "test": "make test",
-    "build": "make build",
-    "prepack": "make build",
-    "postpack": "make clean",
-    "prepublish": "make build",
-    "postpublish": "make clean"
-  },
-  "engines": {
-    "node": ">= 0.8.x"
+    "build": "npm run build:lib && npm run build:dist && npm run build:css",
+    "build:lib": "tsc",
+    "build:dist": "webpack",
+    "build:css": "cpx 'node_modules/quill/dist/quill.*.css' dist",
+    "watch": "tsc --watch",
```

```suggestion
    "watch": "tsc --watch",
    "demo": "npx superstatic demo",
```

This would add a script to serve the demo page locally with live reloading
using ttps://www.npmjs.com/package/superstatic from Firebase (there are several alternatives to this package)

---

Amazing suggestion. Added, thanks!